samples/javascript/E1_HelloSequence/function.json

```diff
@@ -7,4 +7,4 @@
     }
   ],
   "disabled": false
```

Comment:
The `disabled` property is only for V1, could you remove references to it?

Reply:
Done!